packages/types/src/messages/sequence-stopped.ts

```diff
@@ -0,0 +1 @@
+export type SequenceStoppedMessageData = { sequenceError?: unknown }
```

Comment:
Hmm... we should have a separate message for this situation.

```ts
type SequenceErrorMessageData = { error: Error, lifecycle: "init"|"exec"|"handshake"|"call"|"runtime"|"exit"|"cleanup" }
```

Reply:
here I meant only to map the runtime reality. I'd leave a refactor for another PR or.. never xd

Reply:
@alicja-gruzdz let's put it in TD?